docs/course_materials/variables_data_types.html

```diff
@@ -494,11 +494,17 @@ <h3><b>Interactive Input</b></h3>
               <li>input()</li>
               <li>input(prompt) : prints prompt before reading input</li>
             </ul>
-            <li>The result is of String type</li>
           </ul>
           <pre><code data-trim data-noescape class="hljs">
-name = input('Enter your name: ')
-age = int(input('Enter your age: '))
+>>> name = input('Enter your name: ')
```

Comment:
In python 2 you should use ```raw_input()```,
see https://docs.python.org/2/library/functions.html#input

Reply:
It seems that in python2, raw_input casts automatically to a string, while input needs to enter a value with ''.
Which one is better for your opinion?